index.html

```diff
@@ -10,63 +10,100 @@
 
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
-
-  <title>My App</title>
-
-  <link rel="shortcut icon" sizes="32x32" href="/images/app-icon-32.png">
-
-  <meta name="theme-color" content="#fff">
-  <link rel="manifest" href="/manifest.json">
-
-  <script>
-
-    // setup Polymer options
-    window.Polymer = {lazyRegister: true, dom: 'shadow'};
-
-    // load webcomponents polyfills
-    (function() {
-      if ('registerElement' in document
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1">
+
+    <title>My App</title>
+    <meta name="description" content="My App description">
+
+    <link rel="icon" href="/images/favicon.ico">
```

Comment:
Ok then :)
On Jun 6, 2016 2:12 AM, "Abdón Rodríguez Davila" <notifications@github.com>
wrote:

> In index.html
> <https://github.com/Polymer/app-drawer-template/pull/13#discussion_r65858338>
> :
>
> > -  <script>
> > -
> > -    // setup Polymer options
> > -    window.Polymer = {lazyRegister: true, dom: 'shadow'};
> > -
> > -    // load webcomponents polyfills
> > -    (function() {
> > -      if ('registerElement' in document
> > +  <head>
> > +    <meta charset="utf-8">
> > +    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1">
> > +
> > +    <title>My App</title>
> > +    <meta name="description" content="My App description">
> > +
> > +    <link rel="icon" href="/images/favicon.ico">
>
> @robdodson <https://github.com/robdodson> about the rel, I don't think
> so: https://mathiasbynens.be/notes/rel-shortcut-icon
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Polymer/app-drawer-template/pull/13/files/eccdb268d264ebb8475d934a2a071e6f2582d6fd#r65858338>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABBFDfsU65U7ZB-pzCFQ1doZxFC8GEfhks5qI-SGgaJpZM4ImQJt>
> .
>